extensions/vault/model/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-vault-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-vault-model</artifactId>
+    <name>Quarkus - Vault - Model</name>
+    <description>Store your credentials securely in HashiCorp Vault</description>
```

Comment:
This description should probably explain why this module is separate from the rest

Reply:
Or rather that these are the model classes; 
and the explanation why they are separate could be... in a comment maybe?

Reply:
yes. I focused on the technical part at this point. it has just been copy/pasted. there a few details to polish. thanks.